### PR TITLE
[Dynamic Selection] Modify Dynamic Selection files to clang format

### PR DESCRIPTION
**Pull request by AnuyaWelling2801**
No logic changes.

### CODE REVIEW

**Review by SergeyKopienko — COMMENTED** [review]
See my comment at https://github.com/oneapi-src/oneDPL/pull/1177#issuecomment-1717465035

File: include/oneapi/dpl/dynamic_selection [hunk 1]
```diff
@@ -13,10 +13,11 @@
 #include "oneapi/dpl/internal/common_config.h"
 #include "oneapi/dpl/pstl/onedpl_config.h"
 
-#if _ONEDPL_BACKEND_SYCL != 0 && __INTEL_LLVM_COMPILER >= 20230000 //TODO: Use a config macro that detects the presence of host tasks
-    #define _DS_BACKEND_SYCL 1
+#if _ONEDPL_BACKEND_SYCL != 0 &&                                                                                       \
+    __INTEL_LLVM_COMPILER >= 20230000 //TODO: Use a config macro that detects the presence of host tasks
```
[the remark this answers is not in the document]
Comment (vossmjp) [reply include/oneapi/dpl/dynamic_selection:17]: will do

**Review by vossmjp — APPROVED** [review]
Looks good to me.

**Review by vossmjp — APPROVED** [review]
Looks good (again).

**Pull request merged**